dataset_template.py

```diff
@@ -23,15 +24,18 @@ def __init__(self, custom_template_file):
 
             :param custom_template_file: file, the template file to be accessed
         """ 
+
+        # Joshua Zahner - 02/2019
```

Comment:
@2gotgrossman Could you provide your opinion on the following code block? I think this is creating a copy of the template file in a new directory, but I don't think that should be a part of this object, as it doesn't deal with the data, but rather file handling. Thoughts?

Reply:
Yes I can, but could you please move it to a separate PR?